<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>netfilter</class>
 <widget class="QWidget" name="netfilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>netfilter</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>端口号:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IPLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>86</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="PORTLineEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>NETFILTER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_IP_Btn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>添加IP地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Del_IP_Btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>删除IP地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Display_IP_Btn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>显示过滤的IP地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_PORT_Btn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>添加端口号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Del_PORT_Btn">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>190</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>删除端口号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Disay_PORT_Btn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>显示过滤的端口号</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textview">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>511</width>
     <height>78</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
